app/__init__.py

```diff
@@ -28,7 +33,12 @@ def create_app(test_config=None):
 
     db.init_app(app)
     migrate.init_app(app, db)
+    from app.models.task import Task
+    from app.models.goal import Goal
 
     # Register Blueprints here
+    from .routes import task_list_bp,goals_bp
+    app.register_blueprint(task_list_bp)
+    app.register_blueprint(goals_bp)
 
     return app
```

Comment:
Remember to make an empty init.py file in any package folder/subfolder. app has one, but we should have one here in the models folder as well.